webrtc.html

```diff
@@ -7520,6 +7520,12 @@ <h3>Simulcast functionality</h3>
         implications of this model is that the <code>addTrack</code> method cannot provide simulcast
         functionality since it does not take <code>sendEncodings</code> as an argument, and therefore cannot
         configure an <code><a>RTCRtpTransceiver</a></code> to send simulcast.</p>
+        <p>The answerer cannot set the <a>simulcast envelope</a> directly. Upon calling the 
+        <code>setRemoteDescription</code> method of the <code><a>RTCPeerConnection</a></code> object, 
+        the <a>simulcast envelope</a> should be configured on the <code><a>RTCRtpTransceiver</a></code>
+        based upon the specified <code><a>RTCSessionDescription</a></code>. While layers cannot be removed,
+        they can be marked as inactive by setting the <code>active</code> attribute to <code>false</code> 
+        effectively disabling the layer.</p>
```

Comment:
Can you update the setRemoteDescription() steps to make it explicitly part of the algorithms how the encodings is set?

Reply:
I.e. clarify the "Create an RTCRtpSender, sender, from the media description." at https://w3c.github.io/webrtc-pc/#set-description

Reply:
Done. please let me know if the added steps are clear enough.